notebooks/day2/03-fluky_fields.jl

```diff
@@ -901,7 +895,7 @@ version = "2.1.7"
 [[Statistics]]
 deps = ["LinearAlgebra", "SparseArrays"]
 uuid = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
-version = "1.10.0"
+version = "1.9.0"
```

Comment:
@MichielStock , best je julia versie naar 1.10.0 brengen anders gaan we heel de tijd dit heen en weer gebox krijgen van dependencies